pkg/actors/actors_test.go

```diff
@@ -790,6 +791,43 @@ func TestOverrideTimerCancelsMultipleActiveTimers(t *testing.T) {
 	})
 }
 
+func timerRepeats(ctx context.Context, t *testing.T, repeats uint32) {
+	requestC := make(chan testRequest, 10)
+	appChannel := mockAppChannel{
+		requestC: requestC,
+	}
+	testActorsRuntime := newTestActorsRuntimeWithMock(&appChannel)
+	actorType, actorID := getTestActorTypeAndID()
+	fakeCallAndActivateActor(testActorsRuntime, actorType, actorID)
+
+	timer := createTimerData(actorID, actorType, "timer", "2s", "2s", repeats, "callback", "data")
+	err := testActorsRuntime.CreateTimer(ctx, &timer)
+	assert.Nil(t, err)
+
+	var cnt uint32
+LOOP:
```

Comment:
I really do not see LOOP in tests before, is it common, could we use other way to test this?

WDYT? /cc @artursouza @yaron2

Reply:
What's wrong with the "LOOP"? How else would you recommend breaking for/select loop?
It is not related to the tests, but rather one of the golang coding practices.

Reply:
It would be best If we can use `channel`. I do not like LOOP personlly.

Reply:
Personal preferences aside, do you see any concrete issue with using loop labels?
AFAIK, this is an accepted coding practice.

Reply:
I cannot find the code where it was being used. I would personally avoid using labels with "goto"-like syntax. If there is a linter to catch these, I would be all in to put that into the code.

Reply:
Like I said, loop labels are valid and necessary Go syntax.
Here is an example of using it in Kubernetes:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/plugin/pkg/audit/buffered/buffered.go#L200